app/code/Magento/Backend/Block/Widget/Button/ButtonList.php

```diff
@@ -127,8 +127,8 @@ public function getItems()
      */
     public function sortButtons(Item $itemA, Item $itemB)
     {
-        $sortOrderA = intval($itemA->getSortOrder());
-        $sortOrderB = intval($itemB->getSortOrder());
+        $sortOrderA = (int) $itemA->getSortOrder();
```

Comment:
Please do an amend commit removing spaces, i.e. `(int) $itemA->getSortOrder()` -> `(int)$itemA->getSortOrder()`.

Reply:
Will do it on our side as `php-cs-fixer` is able to do it.

Reply:
@orlangur so i do nothing right :-)

Reply:
@mhauri do nothing please, sorry for confusing)